src/main/target/SPRACINGF3/target.h

```diff
@@ -54,6 +54,9 @@
 #define USE_MAG_AK8975
 #define USE_MAG_HMC5883
 #define MAG_HMC5883_ALIGN CW270_DEG
+// test for ist8310 , ucomment it
```

Comment:
I would expect that autodetection can handle this(HMC should be tried sooner). And there is config option to select magnetometer directly ..